test/functional/README.md

```diff
@@ -0,0 +1,376 @@
+## Functional test suite
+
+This directory contains the functional test suite for the IoTA Node Lib. This test suite is based on mocha and chai. For
+mocks, we use the nock library. Additionally, it uses some specific functions to ease to implement the test. Helper
+functions are located in the `testUtils.js` file.
+
+There are 2 tests files in this directory:
+
+-   `fuctional-tests.js`: This file contains the test defined in the "classic way". This means, coded in the JS file as
+    any other mocha test. It uses the functions defined in the `testUtils.js` file to simplify the tests.
+-   `functional-tests-runner.js`: This file contains the test runner that executes the tests defined as JSON in a
+    separate file (`testCases.js`) in a "automatic way". This file is loaded by the test suite and the test cases are
+    automatically executed.
+
+The recommended way is to use `testCases.js` (run by `functional-tests-runner.js`). The `fuctional-tests.js` file is
+provides as basically as an example in the case the "old way" needs to be used (I.E: when the test follow a different
+pattern than the one supported by the test runner).
+
+### Automatic test cases
+
+Each test case is defined as a JSON object in the `testCases.js` file. This file is loaded by the test suite and the
+test cases are automatically generated. Each test case is defined as an object with the following elements:
+
+-   `describeName`: The name of the `DESCRIBE` test case. This will be used to generate the test case name in the mocha
```

Comment:
```suggestion
-   `describeName`: The name of the `DESCRIBE` test case. This will be used to generate the test case name in the mocha. Note this name is prefixed by a pure number (e.g `0010 - Simple group without attributes`) which specifies the group to which the test belong (usually meaning a feature) or by a number preced by the `#` symbol to refer to an issue number (e.g. `#1234 - Bug foobar`).
```